File: LeetCode/Easy/2652-sum-multiples/2652-sum-multiples.cpp
class Solution {
public:
    int sumOfMultiples(int n) {
        int ret = 0;
        
        for (int i = 3; i < n + 1; i += 3) ret += i;
        for (int i = 5; i < n + 1; i += 5)
            if (i % 3 != 0) ret += i;
        for (int i = 7; i < n + 1; i += 7)
            if (i % 3 != 0 && i % 5 != 0) ret += i;
        
        return ret;
    }
};

File: LeetCode/Easy/2652-sum-multiples/2652-sum-multiples.py
class Solution:
    def sumOfMultiples(self, n: int) -> int:
        ret = 0
        
        for i in range(3, n + 1, 3):
            ret += i

        for i in range(5, n + 1, 5):
            if i % 3 != 0:
                ret += i 
        
        for i in range(7, n + 1, 7):
            if i % 3 != 0 and i % 5 != 0:
                ret += i

        return ret